app/javascript/features/lobby/Lobby.jsx

```diff
@@ -90,6 +93,18 @@ export const Lobby = () => {
         dispatch(updateBattle({ battle: message.battle }))
         break
       }
```

Comment:
Please see https://github.com/mma127/omg_rails/pull/85/files#diff-34e65b1d905f98b49ec22bcc6dcd86c5e478021004cee21553791a0dfd7daacbR30, we need to use the `useRef` hook to be sure that we can push the most up to date `player` data into this method. Additionally, we need to dispatch `fetchActiveBattles` so that we can pull the latest battle state to make sure actions like create battle are properly disabled when the player joins a battle.